plugins/epos/plugin.py

```diff
@@ -384,37 +388,120 @@ def rda_a1_03d(self):
 
         return (points, msg_list)
 
-    def rda_i1_02m(self):
-        """ Indicator RDA-A1-01M
-        This indicator is linked to the following principle: I1: (Meta)data use a formal, accessible,
-        shared, and broadly applicable language for knowledge representation. More information
-        about that principle can be found here.
+    def rda_a1_04m(self):
+        """Indicator RDA-A1-01M
+        This indicator is linked to the following principle: A1: (Meta)data are retrievable by their
+        identifier using a standardised communication protocol. More information about that
+        principle can be found here.
+        The indicator concerns the protocol through which the metadata is accessed and requires
+        the protocol to be defined in a standard.
+        Technical proposal:
+        Parameters
+        ----------
+        item_id : str
+            Digital Object identifier, which can be a generic one (DOI, PID), or an internal (e.g. an
+            identifier from the repo)
+        Returns
+        -------
+        points
+            A number between 0 and 100 to indicate how well this indicator is supported
+        msg
+            Message with the results or recommendations to improve this indicator
+        """
+        points = 0
+        msg = "Metadata can not be found"
+        print(self.oai_base)
+
+        parsed_endpoint = urllib.parse.urlparse(self.oai_base)
+        protocol = parsed_endpoint[0]
```

Comment:
I think it is better to use the `scheme` attribute of urlparse:

`protocol = parsed_endpoint.scheme`